### PR TITLE
SEG-26: [SPM] Add Support for Swift Package Manager

### LINKED ISSUE #25: Migrate to XCode12 and run iOS14
**Description**
This is the epic to group all the issues for a smooth migration of the project to Xcode 12 and iOS14.

As a Nedap apps developer
I want the project to build in Xcode 12 and run on iOS14
So that we can use the latest tools

**Rules/Acceptance criteria**
* R1: Build in Xcode 12 and run on iOS14.
* R2: No issues should be spotted.

**Is the issue ready to start development?**
- [X] It meets our [DoR](https://github.com/nedap/healthcare-mobile/wiki/DoR).

### PR DESCRIPTION
**Pull request by dregatos**
**Context**
- Why do we want/need it?

DAVID 6 - SPM 0

**Relevant issue**
- Closes #25 

**Changes**
- Added support for SPM.
- We also still support Carthage and Cocoapods.

The new release, v1.1.6, will be created when this PR is approved.

**Known problems**
- None

**Extra attention**
- What do I need to pay extra attention to as a reviewer?

**Check-list**
- [ ] Acceptance criteria described in the issue are satisfied, or the bug(s) is/are fixed.
- [ ] Documentation present (relevant information is properly documented).
- [ ] Written tests for test suite (optional).
- [ ] CHANGELOG updated.

**Tested on devices**
- [ ] Minimum supported iOS version.
- [ ] Maximum supported iOS version.
- [ ] Lowest resolution supported.
- [ ] Highest resolution supported.
- [ ] Device with a notch.

**Is this issue considered as Done?**
- [X] It meets our [DoD](https://github.com/nedap/healthcare-mobile/wiki/DoD).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Segnify for iOS
 
-[![Swift version](https://img.shields.io/badge/swift-5-brightgreen.svg)](https://img.shields.io/badge/swift-5-brightgreen.svg)
+[![Swift version](https://img.shields.io/badge/swift-5.3-brightgreen.svg)](https://img.shields.io/badge/swift-5-brightgreen.svg)
+[![Platform](https://img.shields.io/cocoapods/p/Segnify.svg)](https://img.shields.io/cocoapods/p/Segnify.svg)
+![Swift Package Manager](https://img.shields.io/badge/SPM-supported-DE5C43.svg?style=flat)
```
Comment (barthopster) [inline README.md:5]: What about applying the style of Carthage to the SPM badge?

```suggestion
![Swift Package Manager](https://img.shields.io/badge/SPM-compatible-4BC51D.svg?style=flat)
```

**Review by barthopster — CHANGES_REQUESTED** [review]
Looking great @dregatos, thanks for taking the effort 👏🏼 

Two very small comments, for the rest it's working as expected.

File: README.md (hunk 1 above)
Comment (dregatos) [reply README.md]: Good idea.

**Review by barthopster — APPROVED** [review]

**Pull request merged**